openmdao/core/tests/test_group.py

```diff
@@ -71,6 +71,28 @@ def setup(self):
     def compute(self, inputs, outputs):
         self._order_list.append(self.pathname)
 
+class TestSubsystemConfigError(unittest.TestCase):
+
+    def test_add_subsystem_error_on_config(self):
+        class SimpleGroup(om.Group):
+
+            def __init__(self):
+                super(SimpleGroup, self).__init__()
```

Comment:
this should prob be `setup` rather than `__init__`

Reply:
Fixed